setup.py

```diff
@@ -42,9 +42,9 @@ def run_tests(self):
 packages = ['requests']
 
 requires = [
-    'chardet>=3.0.2,<3.1.0',
-    'idna>=2.5,<2.9',
-    'urllib3>=1.21.1,<1.26,!=1.25.0,!=1.25.1',
+    'chardet>=3.0.2,<4',
+    'idna>=2.5,<3',
+    'urllib3>=1.21.1,<2,!=1.25.0,!=1.25.1',
```

Comment:
I agree with the minor dependencies, however probably best to leave the `urllib3` version locks. Requests and urllib3 coordinate releases usually so the version locks are safe. :)

Reply:
If releases are co-ordinated, surely there's no need for a lockstep version pin?

Reply:
I agree with Seth that the maintainers should be managing the `urllib3` version bump.

Requests digs into the internals of urllib3 in a handful of places. While urllib3 may release a non-breaking change for external APIs, those changes have broken Requests in the past. There have also been cases where a release of urllib3 introduces a non-trivial bug that requires a few weeks to fix. For that reason we usually wait a few days to a week to do the official bump.

Reply:
Okay, any reason not to just specify `urllib3>=1.25,<1.26`?

Reply:
The != declarations were made because the first couple releases of urllib3 1.25 had breaking changes. We'll clean those up once the next minor version is out and we bump the pin to < 1.27.

Reply:
Sure, but can you think of a reason why anyone would need, or indeed want, to stick to urllib3 1.21.1 to 1.24.x?

Reply:
We’ve left the wider gap open since individuals may not have complete control over their ecosystem and what version of urllib3 that’s being vended.

Eventually we’ll need to start pulling the tail up, but we haven’t really talked about when to do that yet. I’d say we’d want to support at least 3 trailing minor versions. Let’s just leave it for now.